### PR TITLE
Remove mctaylors' version of the Russian language

### PR DESCRIPTION
**Pull request by mctaylors**
295 PR/issues ~(not 300, however)~ or ~1.5 years ago, I made #2, the Russian language replacement aka mctaylors-ru. This was my first contribution to the Octobot project (formerly known as Boyfriend). This was to add some sort of unique, unusual feature to Octobot, which doesn't have any moderator bots. Everyone loved the language. But it just became difficult to maintain. I certainly don't want to get rid of it, but it leaves me no other choice. This isn't a joke or anything like that. I'm tired of maintaining it. And I'm sure the other contributors are too.

This PR removes the mctaylors-ru language.

### CODE REVIEW

File: src/Data/Options/LanguageOption.cs
Comment (Octol1ttle) [inline src/Data/Options/LanguageOption.cs:1]: what happens to configs that are using `mctaylors-ru`?

Comment (mctaylors) [reply src/Data/Options/LanguageOption.cs:1]: complete bot malfunction in a guild that uses `mctaylors-ru`. who uses `mctaylors-ru` anyway

Comment (Octol1ttle) [reply src/Data/Options/LanguageOption.cs:1]: yeah that's not acceptable

**Review by Octol1ttle — CHANGES_REQUESTED** [review]

File: src/Responders/GuildLoadedResponder.cs [hunk 1]
```diff
@@ -94,6 +96,18 @@ public async Task<Result> RespondAsync(IGuildCreate gatewayEvent, CancellationTo
...
+    {
+        var settings = data.Settings;
+
+        if (GuildSettings.Language.Get(settings).Name is "tt-RU")
+        {
+            GuildSettings.Language.Set(settings, "ru");
+        }
+
+        return Task.CompletedTask;
+    }
```
Comment (Octol1ttle) [inline src/Responders/GuildLoadedResponder.cs:109]: This belongs in GuildDataService#MigrateGuildData

File: src/Services/GuildDataService.cs [hunk 2]
```diff
@@ -167,6 +167,14 @@ private void MigrateGuildData(Snowflake guildId, string newPath)
             _logger.LogInformation("Moved guild data to separate folder: \"{OldPath}\" -> \"{NewPath}\"", oldPath,
                 newPath);
         }
+
+        var settings = (await GetData(guildId, ct)).Settings;
```
Comment (Octol1ttle) [inline src/Services/GuildDataService.cs:171]: and that's a StackOverflowError

**Review by Octol1ttle — CHANGES_REQUESTED** [review]

**Comment by mctaylors** [comment]
It's fun to realize that I'm about to hit squash and merge and my whole language will be normal `ru`. I will also need to edit the wiki, btw.

**Pull request merged**